lib/transport/selenium-webdriver/service-builders/selenium.js

```diff
@@ -19,6 +19,8 @@ class SeleniumServer4 extends DriverService {
     let java = 'java';
     if (process.env['JAVA_HOME']) {
       java = path.join(process.env['JAVA_HOME'], 'bin/java');
+    } else {
+      throw new Error('Java is not installed');
```

Comment:
Again, you seem to have not tested it. Not having a `JAVA_HOME` env variable does not necessarily mean that JAVA is not installed in the system.

At this point, you are spamming the project with your PRs more than helping out with the issues. This is not the behaviour we encourage here.

Whatever PRs you raise and mark for review should be properly tested. And if you don't understand anything, ask questions but at least try to show that you care about actually solving the issue and not just be the first one to create a PR. This is not a race going on over here.